### DISCUSSION TITLE
Is Copilot still available in Student Developer Pack?

### DISCUSSION THREAD

**Discussion by hellohawaii** [community/community#65433] [GitHub Education] [answered]
### Hi everyone,

I was suddenly notified by my Copilot plugin that _GitHub Copilot could not connect to server. Extension activation failed: "No access to GitHub Copilot found. You are currently logged in as USERNAME."_. I opened the Github website, and in the setting->Copilot,  I can only see a "start trial" button, without my subscription.
<img src="https://github.com/community/community/assets/37056530/a7b0482e-16de-4856-a46b-83b0a89ef667" width=500px>

I have been using Copilot for free as a student. I am currently a university student in China. I wonder whether Copilot is still available in the Student Developer Pack in China? 

May be related: I received an email days before, asking me to re-verify my academic affiliation. I submitted a request yesterday and was approved. I know it takes several days to get access to the benefits after the approval, is this the reason? However, my previous student identity should be valid until September 10, as indicated in the email I received, am I correct?
<img src="https://github.com/community/community/assets/37056530/16ddde61-20e5-4f51-a910-afabe031c7b2" width=500px>

**Comment by arafajs** [comment, marked as answer]
it usually takes 72 hours for the benefits to be reactivated after your application is approved. However, during busy periods like the start of a new semester, it may take a bit longer.

**Reply by hellohawaii** [reply]
Thank you for your reply. So although the "Coupon for GitHub Pro" expires on Sep 10 with 14 extra days, the GitHub for Education expired earlier? I checked the GitHub Education website again and in the "Benefits" I found a recording that says "Expired Expired Aug 27, 2023". It is quite strange since I was able to use Copilot yesterday (Aug 29).
